make/connection_crd/main.go

```diff
@@ -0,0 +1,13 @@
+package main
+
+import (
+	"fmt"
+
+	crd "github.com/jetstack/venafi-connection-lib/config/crd/bases"
+)
+
+// With this tool, we no longer have to use something like `helm template` to
+// pull the CRD manifest from the venafi-connection-lib project.
+func main() {
+	fmt.Print(string(crd.VenafiConnectionCrd))
+}
```

Comment:
@maelvls you can take a look at https://github.com/jetstack/venafi-enhanced-issuer which has some logic for generating and modifying these Venafi Connection CRDs.